bentoml/server/marshal_server.py

```diff
@@ -115,16 +70,34 @@ def load_config(self):
         for key, value in gunicorn_config.items():
             self.cfg.set(key.lower(), value)
 
+    def setup_prometheus_multiproc_dir(self):
```

Comment:
move this function out and share the code with the one in the gunicorn API server?